test/data/mock-data.js

```diff
@@ -429,6 +429,291 @@ const TRADES_API_MOCK_RESULT = [
   },
 ];
 
+const SWAP_TRANSACTION_TRADES_QUOTE_MOCK = [
```

Comment:
I think it can be good if we add a more explicit name, so it can be re-used in another test if needed. Something like:

```suggestion
const SWAP_ETH_USDC_TRADES_MOCK = [
```